apps/src/music/redux/musicRedux.ts

```diff
@@ -76,7 +76,7 @@ const initialState: MusicState = {
   playbackEvents: [],
   orderedFunctions: [],
   lastMeasure: 0,
-  soundLoadingProgress: 0,
+  soundLoadingProgress: 1,
```

Comment:
Curious whether future visitors to the code will understand why this starts at 1?

Reply:
Good call, will add a comment